src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -661,6 +664,31 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         e.Handled(handled);
     }
 
+    void TermControl::_SendKeyEvent()

```

Comment:
Feel a bit weird about a void method wrapping a bool one and discarding the result